doc/parted.md

```diff
@@ -0,0 +1,71 @@
+# parted And Partition Alignment
+
+## General
+
+- parted will try to align newly created partitions. But only if the partition
+start/end are a bit 'fuzzy'. For example:
+
+ - 'parted p ext2 1% 2%' will give nicely aligned partitions; so does
+ - 'parted p ext2 1MB 2MB'. But the supposedly more correct
+ - 'parted p ext2 1MiB 2MiB' will **not** do what you (well, at least I did) expect. The partition size will be 1 MiB + 1 sector.
+ 
+- parted does **not** align to cylinder sizes
+
+- parted will align to 1MiB, unless your the underlying raid config does
+have a chunk size which is not a power of 2 smaller than 1MiB. In that case parted aligns to
+chunk size * 'usable disks' - see details below.
```

Comment:
You should mention that parted uses topology information provided by the kernel or libblkid (see http://git.savannah.gnu.org/cgit/parted.git/tree/libparted/arch/linux.c#n3206). The alignment_offset and optimal_io_size are also available under /sys.